### PR TITLE
Plastic flaps are once again immune to being hit in melee

### PR DESCRIPTION
**Pull request by ChangelingRain**
They're also very resistant to most other damage.

### CODE REVIEW

**Comment by Iamgoofball** [comment]
why are plastic flaps immune to everything anyways

**Comment by WJohn** [comment]
A bunch of places were designed around them being invincible. Suddenly being able to break into them easily breaks intended balance, and would require some remapping at every mulebot destination.

**Comment by jughu** [comment]
Who even uses mulebots these days

**Comment by KingPhilipIII** [comment]
People who hack them to cause chaos. Or Nukeops with a sentience potion and a really good sense of humor. (I've had a sentient mulebot save me by running an officer that was batoning over.)

**Comment by ExcessiveUseOfCobblestone** [comment]
so in other words the only time they have use is when they aren't being used to transfer items?

**Comment by KingPhilipIII** [comment]
Well. With the ability to shuttle through disposals, which is faster, less prone to interception and easier, pretty much.

File: code/game/objects/structures/plasticflaps.dm [hunk 1]
```diff
@@ -1,8 +1,9 @@
-/obj/structure/plasticflaps	//HOW DO YOU CALL THOSE THINGS ANYWAY
+/obj/structure/plasticflaps
 	name = "plastic flaps"
 	desc = "Definitely can't get past those. No way."
-	icon = 'icons/obj/stationobjs.dmi'	//Change this.
+	icon = 'icons/obj/stationobjs.dmi'
 	icon_state = "plasticflaps"
+	armor = list(melee = 100, bullet = 80, laser = 80, energy = 100, bomb = 50, bio = 100, rad = 100, fire = 50, acid = 50)
```
Comment (phil235) [inline code/game/objects/structures/plasticflaps.dm]: this means blobs and xenos can't destroy them though.

Comment (ChangelingRain) [inline code/game/objects/structures/plasticflaps.dm]: Well THANKS, phil!
They need to be immune to melee so you can't just hit them with a wrench, aliens can already go through them, and well I guess I'll have to override blob_act() for these, then.

**Comment by Tacolizard** [comment]
i use mulebots because they return crates

File: code/game/objects/structures/plasticflaps.dm (hunk 1 above)
Comment (PKPenguin321) [inline code/game/objects/structures/plasticflaps.dm]: Place your core under flaps new meta

Comment (Supermichael777) [inline code/game/objects/structures/plasticflaps.dm]: yeah that would be problematic

**Pull request merged**